task/git-clone/0.1/git-clone.yaml

```diff
@@ -165,6 +172,18 @@ spec:
         fi
       }
 
+      check_symlinks() {
+        find $CHECKOUT_DIR -type l -print | \
+          while read symlink
+          do
+            target=$(readlink -f "$symlink")
+            if ! [[ "$target" =~ ^$CHECKOUT_DIR ]]; then
+              echo "The cloned repository contains symlink pointing outside of the cloned repository: $symlink"
```

Comment:
would be better to aggregate all symlinks pointing outside and print all of them not just first one.

Reply:
changed